braindecode/models/eegitnet.py

```diff
@@ -271,10 +294,10 @@ def __init__(
         self.add_module(
             "TC_block2",
             _TCBlock(
-                in_ch=14,
-                kernel_length=4,
-                dialation=2,
-                padding=6,
+                in_ch=tcn_in_channel,
+                kernel_length=tcn_kernel_size,
+                dilatation=int(tcn_dilatation * 2),
+                padding=int(tcn_padding * 2),
```

Comment:
```suggestion
                dilatation=tcn_dilatation * 2,
                padding=tcn_padding * 2,
```
it should not be needed if you multiply :) 
Same below